packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeOption.tsx

```diff
@@ -65,7 +77,7 @@ export const TimeRangeOption = memo<Props>(({ value, onSelect, selected = false,
         id={id}
         onChange={() => onSelect(value)}
       />
-      <label className={styles.label} htmlFor={id}>
+      <label className={cx(styles.label, selected && styles.labelSelected)} htmlFor={id}>
```

Comment:
moved selection class to label here as the label has the padding and otherwise the selected state would be outside the element of focus (the input). Very complicated markdown here for a simple list of options :